--- src/Terraria\Terraria.Graphics.Effects\SimpleOverlay.cs
+++ src/tModLoader\Terraria.Graphics.Effects\SimpleOverlay.cs
@@ -43,13 +_,13 @@
 			this._shader.Update(gameTime);
 		}
 
-		internal override void Activate(Vector2 position, params object[] args)
+		public override void Activate(Vector2 position, params object[] args)
 		{
 			this.TargetPosition = position;
 			this.Mode = OverlayMode.FadeIn;
 		}
 
-		internal override void Deactivate(params object[] args)
+		public override void Deactivate(params object[] args)
 		{
 			this.Mode = OverlayMode.FadeOut;
 		}

